data/tokens.json

```diff
@@ -3904,6 +3904,77 @@
         "value": "{gtmds_shadow-md}",
         "type": "boxShadow"
       }
+    },
+    "Canvas Menu": {
+      "gtmds_canvas-menu_shadow-sm": {
+        "value": {
+          "x": "0",
+          "y": "1",
+          "blur": "1",
+          "spread": "0",
+          "color": "#5A7F9E22",
+          "type": "dropShadow"
+        },
+        "type": "boxShadow"
+      },
+      "gtmds_canvas-menu_shadow-md": {
+        "value": {
+          "x": "0",
+          "y": "2",
+          "blur": "6",
+          "spread": "0",
+          "color": "#5A7F9E22",
+          "type": "dropShadow"
+        },
+        "type": "boxShadow"
+      },
+      "gtmds_canvas-menu-item_shadow": {
+        "value": {
+          "x": "0",
+          "y": "1",
+          "blur": "2",
+          "spread": "0",
+          "color": "#5A7F9E4D",
+          "type": "innerShadow"
+        },
+        "type": "boxShadow"
+      },
+      "gtmds_canvas-menu-item_color": {
+        "value": "{gtmds_color_gray-9}",
+        "type": "color"
+      },
+      "gtmds_canvas-menu-item_color-hover": {
+        "value": "{gtmds_color_gray-12}",
+        "type": "color"
+      },
+      "gtmds_canvas-menu-item_background-color": {
+        "value": "{gtmds_color_white}",
+        "type": "color"
+      },
+      "gtmds_canvas-menu-item_background-color-selected": {
+        "value": "{gtmds_color_gray-3}",
+        "type": "color"
+      },
+      "gtmds_canvas-menu_position-offset-lg": {
+        "value": "{gtmds_space-lg}",
+        "type": "spacing"
+      },
+      "gtmds_canvas-menu_position-offset-xl": {
```

Comment:
Should offset be tokenized? Isn't that part of a larger container that is not part of the component itself?

Reply:
I see your point and it gives me better idea, but we will have to keep the tokens as such, just use them differently in Figma, thanks!!!

Reply:
Curious to find one 😆

Reply:
Nothing special, just a frame with paddings that you overlay the canvas with :)

Reply:
Just pushed that so check out the commit below and the file and resolve this if you like it better now